inlanecruising_plugin/include/third_party_library/spline.h

```diff
@@ -0,0 +1,404 @@
+/*
```

Comment:
Just making a note that we still need to resolve this licensing issue. I'm not even sure if we're technically able to host them in the same repo, but we'll have to sort this out with a different library.

Reply:
Yep. It's being tracked here
https://github.com/usdot-fhwa-stol/carma-platform/issues/936